modules/programs/eza.nix

```diff
@@ -64,14 +64,61 @@ with lib;
       '';
     };
 
+    all = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Show hidden and 'dot' files.
+      '';
+    };
+
+    long = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Display extended details and attributes.
+      '';
+    };
+
+    extended = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        List each file’s extended attributes and sizes.
+      '';
+    };
+
+    header = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Add a header row to each column.
+      '';
+    };
+
+    clasiffy = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Display type indicator by file names.
+      '';
+    };
```

Comment:
`--classify` takes auto/always/never, so it would make more sense to turn this into an enum, and then default it to auto

Reply:
Good suggestion. Done.